scripts/benchmark_all_pallets.sh

```diff
@@ -7,27 +7,20 @@ COLLATOR=./target/release/integritee-collator
 
 mkdir -p $INTEGRITEE_RUNTIME_WEIGHT_DIR
 
-pallets=(
-  "frame_system" \
-  "pallet_balances" \
-  "pallet_timestamp" \
-  "pallet_vesting" \
-  "pallet_teerex" \
-  "pallet_claims" \
-  "pallet_migration" \
-  "pallet_multisig" \
-  "pallet_proxy" \
-  "pallet_scheduler" \
-  "pallet_teeracle" \
-  "pallet_treasury" \
-  "pallet_utility" \
-)
+$COLLATOR benchmark pallet \
+    --chain integritee-rococo-local-dev \
+    --list |\
+  tail -n+2 |\
+  cut -d',' -f1 |\
+  uniq > "integritee_runtime_pallets"
 
```

Comment:
haha, what black magic is that?

Reply:
Got fed up with posting every pallet twice (they need to be listed [here](https://github.com/integritee-network/parachain/blob/bh-democracy-pallet/polkadot-parachains/integritee-runtime/src/lib.rs#L800-L816) anyway). Way too error-prone to forget a pallet. And not my magic, but the one from parity: 

https://github.com/paritytech/polkadot/blob/master/scripts/ci/run_benches_for_runtime.sh#L15-L20

Reply:
I agree, good change!